packages/core/admin/admin/src/content-manager/components/PreviewWysiwyg/index.js

```diff
@@ -13,7 +13,7 @@ import sanitizeHtml from './utils/satinizeHtml';
 import Wrapper from './Wrapper';
 
 const PreviewWysiwyg = ({ data }) => {
-  const html = useMemo(() => sanitizeHtml(md.render(data || '')), [data]);
+  const html = useMemo(() => sanitizeHtml(md.render(data.replaceAll('\\n', '\n') || '')), [data]);
```

Comment:
I'm wondering if this applies to `\\n` only, or also other ASCII operators such as `\\r` or `\\t`?

Reply:
Yeah other characters need fixing like `\r` but IMO it's all not markdown and I don't think we should support this. Otherwise, we have to re-add all the characters here. wdyt?

Reply:
Sorry, I didn't know you were waiting for me here. Alright, then let's go with what you have!